README.md

```diff
@@ -84,6 +85,7 @@ Name                                                      | Status       | Short
 [parallel-rustc_stream]: https://rust-lang.zulipchat.com/#narrow/stream/187679-t-compiler.2Fwg-parallel-rustc
 [rfc-2229-stream]: https://rust-lang.zulipchat.com/#narrow/stream/189812-t-compiler.2Fwg-rfc-2229
 [mir-opt-stream]: https://rust-lang.zulipchat.com/#narrow/stream/189540-t-compiler.2Fwg-mir-opt
+[pipelining-stream]: FIXME
```

Comment:
If this all looks good to go, we'll likely want to fill this in!

Reply:
https://rust-lang.zulipchat.com/#narrow/stream/195180-t-compiler.2Fwg-pipelining